src/org/pahappa/systems/registrationapp/views/UserView.java

```diff
@@ -59,21 +66,106 @@ public void displayMenu() {
         }
     }
 
-    private void registerUser() {
+    private void registerUser() throws NoSuchFieldException, ParseException {
+        System.out.print("Enter first name: "); // Prompt user for input
+        String firstName = scanner.nextLine();
+        System.out.print("Enter last name: ");
+        String lastName = scanner.nextLine();
+        System.out.print("Enter username: ");
+        String username = scanner.nextLine();
+        System.out.print("Enter date of birth: ");
+        String dateOfBirth = scanner.nextLine();
```

Comment:
Help the user understand the format of the date they should provide you since your Date formatter is using a specific format.